### PR TITLE
feat: add new sample contact exporter

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: graph-toolkit-contact-exporter/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Contact Exporter sample APP provides an easy way to export your teams contact info to Excel file. This APP is using [Microsoft Graph Toolkit](https://docs.microsoft.com/en-us/graph/toolkit/overview) as UI component.
+
+![Contact Exporter Overview](images/overview.png)
+
+## Prerequisite
+- [NodeJS](https://nodejs.org/en/)
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit)
+- An [Azure subscription](https://azure.microsoft.com/en-us/free/)
```
Comment (adashen) [inline graph-toolkit-contact-exporter/README.md:17]: Can we move the azure requirement to provision/deploy section?

File: graph-toolkit-contact-exporter/tabs/src/components/Tab.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+        clientId: process.env.REACT_APP_CLIENT_ID
+      }
+    });
+    const credential = new TeamsUserCredential();
+
+    this.credential = credential;
+    // Only these two permission can be used without admin approval in microsoft tenant
+    this.scope = [
+      "User.Read",
+      "User.ReadBasic.All",
```
Comment (adashen) [inline graph-toolkit-contact-exporter/tabs/src/components/Tab.js:72]: Seems permission file doesn't have "User.ReadBasic.All"

File: graph-toolkit-contact-exporter/templates/azure/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# ignore ARM template backup folder
+/backup
```
Comment (adashen) [inline graph-toolkit-contact-exporter/templates/azure/.gitignore:2]: do we still have this folder?

File: graph-toolkit-contact-exporter/README.md (hunk 1 above)
Comment (SLdragon) [reply graph-toolkit-contact-exporter/README.md:17]: Sure

File: graph-toolkit-contact-exporter/tabs/src/components/Tab.js (hunk 2 above)
Comment (SLdragon) [reply graph-toolkit-contact-exporter/tabs/src/components/Tab.js:72]: Will add it

File: graph-toolkit-contact-exporter/templates/azure/.gitignore (hunk 3 above)
Comment (SLdragon) [reply graph-toolkit-contact-exporter/templates/azure/.gitignore:2]: No, this is generate by upgrade tool, and can remove it

**Review by adashen — APPROVED** [review]

**Pull request merged**